basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/BaseLTIService.java

```diff
@@ -736,7 +804,7 @@ protected String deleteContentLinkDao(Long key, String siteId, boolean isAdminRo
 		}
 		catch (IdUnusedException ee)
 		{
-			M_log.warn(this + " LTI content="+key+" placement="+tool.getId()+" could not remove page from site=" + siteStr);
+			M_log.warn("LTI content="+key+" placement="+tool.getId()+" could not remove page from site=" + siteStr);
```

Comment:
Earle,  I want to fix all the log patterns at the same time so as to test thoroughly rather than just fixing the ones I touched in the commit because of the "this +" pattern.